src/main/java/net/laboulangerie/laboulangeriemmo/listener/ServerListener.java

```diff
@@ -92,13 +95,13 @@ public void onRiptideHit(EntityDamageByEntityEvent event) {
             return;
         }
 
-        if (!mmoPlayer.getCooldowns().hasUsed(Abilities.DODGING) || mmoPlayer.getCooldowns().getCooldown(Abilities.DODGING) >= 1) // 1 is the duration of the spin attack TODO: Change when real cooldown is set
+        if (!mmoPlayer.getCooldowns().hasUsed("dodging") || !mmoPlayer.getCooldowns().getCooldowns("dodging").stream().filter(x -> x <= 1).findAny().isPresent()) // 1 is the duration of the spin attack
             return;
 
         if (Utils.getAttackDamage(player, player.getInventory().getItemInMainHand()) > 0)
             event.setDamage(Utils.getAttackDamage(player, player.getInventory().getItemInMainHand()));
 
-        event.getEntity().setVelocity(player.getLocation().getDirection().multiply(1));
+        event.getEntity().setVelocity(player.getLocation().getDirection().multiply(5));
```

Comment:
5 seems to be a lot be sure Minecraft doesn't struggle with it as it used to struggle a lot and older PaperMC (1.8.8) used to display warning above 4 velocity